DependentExtensions/Swig/MakeSwig.bat

```diff
@@ -6,7 +6,19 @@ if "%swigPath%"=="" goto :SKIPADDSLASH
 if "%swigPath:~-1%"=="\" goto :SKIPADDSLASH 
 SET swigPath=%swigPath%\
 :SKIPADDSLASH
-del /F /Q SwigOutput\SwigCSharpOutput\*
+
+set swigOutputCsharp=SwigOutput\SwigCSharpOutput
+set swigOutputCplusDLL=SwigOutput\CplusDLLIncludes
+set swigInterface=SwigInterfaceFiles
+set swigFiles=SwigWindowsCSharpSample\SwigTestApp\SwigFiles

```

Comment:
Note that while you are introducing the variables here, the folder names are still hard coded throughout the batch file (f.e. in line 23).